### ISSUE TITLE
adding comments to content type disables content manager

### ISSUE THREAD

**Issue by pguardiario** [VirtusLab-Open-Source/strapi-plugin-comments#207]
Is this normal behavior? I think I might have misunderstood how to set this up.

I don't see anywhere in the docs where it explains how to set up comments for a content type so I'm hoping someone can look at this and tell me if I did it right

![image](https://user-images.githubusercontent.com/58795222/234495114-c7bc6762-1f91-497d-94a3-7b14f6ced823.png)

**Comment by pguardiario** [comment]
also FYI, not selecting any content types results in it not working at all, which I expect is a bug

**Comment by cyp3rius** [comment]
@pguardiario just checked.

> but doing this disables content manager for these types. It's not cealr if that's expected or where to go from here...

It does not happening to me. Can you please elaborate by how the Content Manager is disabled? I'm able to add new items as Pages as well for Content Type Builder I'm able to edit the Collection without any issues if Strapi is not runned in production mode.

> also FYI, not selecting any content types results in it not working at all, which I expect is a bug

That's the translation mistake, the `not` been missed. So works as expected.

**Comment by pguardiario** [comment]
the types I enable comments for no longer show in the content manager 
![image](https://user-images.githubusercontent.com/58795222/234727264-a1ae4bdf-55c6-45cc-8af0-833756b9fb7d.png)

**Comment by cyp3rius** [comment]
That's really odd behavior @pguardiario . Our plugin stores just a "string" reference to selected types like `uid` and does not modify anything outside of plugin scope (it's disallowed and locked).

May I ask you on what settings you're running your Strapi instance?
- `package.json` (at least dependencies part)
- `node`
- set of commands you're using to run it
- server console log
- if possible a screen recording exactly how the issue occurs

**Comment by pguardiario** [comment]
Thanks for the reply, I actually got it fixed but I'm not sure how. Here are some steps I took:
- unset the "enable comments only for" (none were working without it)
- save and restart strapi
- delete the "custom" comment field (apparently I don't need this? It's not clear in docs what this is for)
- add the content type again in "enable comments only for"
- save and restart strapi

hopefully that will help someone. feel free to close this one.
